dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestLogger.kt

```diff
@@ -50,7 +49,7 @@ class DataRequestLogger {
      */
     fun logMessageForSendBulkDataRequestEmail(bulkDataRequestId: String) {
```

Comment:
Funktionsname nicht mehr ganz gültig jetzt.
Man loggt die message jetzt nur noch dafür, dass das Senden einer Bulk-Data-Request-Mail in der message-queue hitnerlegt wurde.
Ob diese Email jemals gesendet wird, ist hier ja noch unklar.

Reply:
Bereits erledigt. Unklar ist es zum aktuellen Stand aber gar nicht. Emails werden immer direkt verschifft.

Reply:
Doch es ist doch unklar.  Wenn mailjet z.B. down ist, würde man hier doch gar nichts davon mitbekommen und munter loggen "Email is being sent"

Reply:
Ok. Aber in dem Szenario nicht falscher als vorher.

Reply:
Am Telefon besprochen.

Reply:
Aber war ja auch schon erledigt, oder muss da noch was gemacht werden?